src/modules/AutoInstallRules.rb

```diff
@@ -923,7 +923,8 @@ def Merge(result_profile)
             message = _(
               "The XML parser reported an error while parsing the autoyast profile. The error message is:\n"
             )
-            Popup.Error(message + XML.XMLError)
+            message += XML.XMLError
+            Popup.show(message, headline: :error)
```

Comment:
You might have to require `"yast2/popup"` (perhaps it is included by someone else, but still).

Reply:
Ups, It was even the `Yast::Popup` module instead of `Yast2::Popup`

**Fixed**